### PR TITLE
bugfix/trivial: fix update node announcement feature bit add /remove

### PR DESCRIPTION
**Pull request by carlaKC**
## Change Description
Splitting this bugfix out from #7168. 

## Steps to Test
Steps for reviewers to follow to test the change.

### Testing
- [ ] Your PR passes all CI checks.
- [ ] Tests covering the positive and negative (error paths) are included.
- [ ] Bug fixes contain tests triggering the bug to prevent regressions.

### Code Style and Documentation
- [ ] The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation) guidelines, and lines wrap at 80.
- [ ] Commits follow the [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStruacture).
- [ ] Any new logging statements use an appropriate subsystem and logging level.
- [ ]  [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or `[skip ci]` in the commit message for small changes.

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.

### CODE REVIEW

**Review by guggero — APPROVED** [review]
LGTM, thanks for the fix :tada:

**Review by yyforyongyu — APPROVED** [review]
LGTM👍 Thanks for the PR!

File: docs/release-notes/release-notes-0.16.0.md [hunk 1]
```diff
@@ -130,6 +130,10 @@ current gossip sync query status.
 * [A bug has been fixed which could cause `lnd` to crash when parsing a
   malformed HTLC intercept message](https://github.com/lightningnetwork/lnd/pull/7392).
 
+* [A bug] (https://github.com/lightningnetwork/lnd/pull/7408) in the
```
[the remark this answers is not in the document]
Comment (GeorgeTsagk) [reply docs/release-notes/release-notes-0.16.0.md:133]: :point_up_2: Yep, MD won't render it as link

**Review by GeorgeTsagk — APPROVED** [review]
LGTM :lady_beetle:

File: docs/release-notes/release-notes-0.16.0.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (carlaKC) [reply docs/release-notes/release-notes-0.16.0.md]: Oops, updated!

**Pull request merged**